### DISCUSSION TITLE
Why returned products do not increase stock

### DISCUSSION THREAD

**Discussion by 1357310795** [saleor/saleor#16548] [Q&A] [answered]
I wonder what's the semantics of “return product”? I expect "the returned products" returned to stock, but it didn't.

example below:

A fully fulfilled order.
![image](https://github.com/user-attachments/assets/d6726b45-1147-4146-a86a-78913674c152)

Check the stock of apple juice, 267.
![image](https://github.com/user-attachments/assets/2b36f795-1df5-4ec5-9196-caf68a82aeec)

Now return the products.
![image](https://github.com/user-attachments/assets/d5671e4b-6734-44eb-9fd5-6c907f8c9b5b)

Stock is still 267.
![image](https://github.com/user-attachments/assets/69c69461-8d38-4038-9957-fe8ab2b8477d)

***
version: 
dashboard v3.18.2
core v3.18.14

**Comment by IKarbowiak** [comment]
Hi @1357310795,
Thank you for reporting an issue. It looks like a bug. I transformed it into an issue: https://github.com/saleor/saleor/issues/16667

**Comment by IKarbowiak** [comment, marked as answer]
Hi @1357310795,
I'm coming back to you, as @patrys clarified that by default, stock is not restored as the products are assumed to be non-sellable (used items, damaged goods, etc.). So we are missing an option to restock.

**Reply by 1357310795** [reply]
Sorry for my late response. Hope the option to restock will be added soon! 🥰
